Singer.API/ClientApp/package.json

```diff
@@ -52,11 +52,11 @@
       "karma-coverage-istanbul-reporter": "~1.4.2",
       "karma-jasmine": "~1.1.1",
       "karma-jasmine-html-reporter": "^0.2.2",
-      "node-sass": "^4.12.0",
       "standard-version": "^7.0.0",
       "typescript": "~3.2.4"
    },
    "optionalDependencies": {
+      "node-sass": "^4.9.3",
```

Comment:
What happened here? It causes the build to break.

Reply:
Thanks for fixing :)

Reply:
Thanks for fixing :)